src/scene/materials/standard-material-options-builder.js

```diff
@@ -235,7 +235,7 @@ class StandardMaterialOptionsBuilder {
         options.sheenEncoding = stdMat.sheenEncoding || 'linear';
         options.aoMapUv = stdMat.aoUvSet; // backwards compatibility
         options.aoDetail = !!stdMat.aoMap;
-        options.diffuseDetail = !!stdMat.diffuseMap;
+        options.diffuseDetail = !!stdMat.diffuseDetailMap;
```

Comment:
This looks incongruous with the surrounding lines. I'd expect this to be `!!stdMap.diffuseMap` based on the other lines.

Reply:
I will refactor (fix) the other lines as I get to those chunks.